README.md

```diff
@@ -2,16 +2,36 @@
 
 *A proxy re-encryption network to empower privacy in decentralized systems*
 
-![](https://img.shields.io/pypi/pyversions/nucypher.svg)
-![](https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master)
-![](https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg)
-![](https://img.shields.io/discord/411401661714792449.svg)
-[![Documentation Status](https://readthedocs.org/projects/nucypher/badge/?version=latest)](https://nucypher.readthedocs.io/en/latest/?badge=latest)
-![](https://img.shields.io/pypi/l/nucypher.svg)
+[![pypi](https://img.shields.io/pypi/v/nucypher.svg?style=flat)](https://pypi.org/project/nucypher/)
+[![pyversions](https://img.shields.io/pypi/pyversions/nucypher.svg)](https://pypi.org/project/nucypher/)
+[![coveralls](https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master)](https://coveralls.io/github/nucypher/nucypher?branch=master)
+[![circleci](https://img.shields.io/circleci/project/github/nucypher/nucypher.svg?logo=circleci)](https://circleci.com/gh/nucypher/nucypher/tree/master)
+[![discord](https://img.shields.io/discord/411401661714792449.svg)](https://discord.gg/7rmXa3S)
+[![Documentation Status](https://readthedocs.org/projects/nucypher/badge/?version=latest)](https://nucypher.readthedocs.io/en/latest/)
+[![license](https://img.shields.io/pypi/l/nucypher.svg)](https://www.gnu.org/licenses/gpl-3.0.html)
 
 ----
 
-![](https://img.shields.io/pypi/v/nucypher.svg?style=flat)
+The NuCypher network uses the [Umbral](https://github.com/nucypher/pyUmbral) threshold proxy re-encryption scheme to provide cryptographic access controls for distributed apps and protocols.
+
+01. Alice, the data owner, grants access to her encrypted data to 
+anyone she wants by creating a policy and uploading it to 
+the NuCypher network.
+
+02. Anyone can encrypt data using Alice's policy public key. 
```

Comment:
Perhaps we should refine this proposed sentence, since it only gives examples of "small" devices. 
`Examples of entities that can do this are IoT devices, a friend, or Alice herself.`

In this case we're covering both machines and humans. The "a friend" example is weak, but I don't have another idea to convey the meaning of a third-party human entity.

Reply:
Good point - 👍
I agree "friend" is a little weird, but I can't think of a better word 😅- "associate", "collaborator" (?) - clearly still just synonyms of "friend" but perhaps comes across better...

Reply:
"comrade"!

Reply:
I would rephrase the whole of line 21 as:
`Using her policy's public key, any entity can encrypt data on Alice's behalf. This entity could be an IOT device in her car, a collaborator assigned the task of writing data to her policy, or even a third-party creating data that belongs to her – for example, a lab analyzing medical tests.`

Reply:
@arjunhassard Sorry, we merged this PR and forgot to address this comment! I'm all in for this propose change. How about you change it in #704?